src/views/Schedule2.vue

```diff
@@ -53,6 +65,23 @@ export default Vue.extend({
         top: item.top * this.ratio + 10 + 'px',
         left: this.width * index + 10 + 'px'
       };
+    },
+    updateNowBarOffset() {
+      const v = this as any;
+      const firstTalk = v.combinedSchedule.flatMap((array: UiScheduleEvent[][]) => array)
+        .sort((a: UiScheduleEvent, b: UiScheduleEvent) => moment(a.toTime).diff(moment(b.toTime)))[0];
+      const now = moment(new Date().toLocaleTimeString(), 'HH:mm');
```

Comment:
```suggestion
      const now = moment().format('HH:mm');
```

Reply:
We need Moment objects and not strings to use diff()